#92 feature:  A buyer should be able to checkout and pay

oliviertech212 opened the pull request:
## What does this PR do?
- a buyer should be able to checkout and pay
## Description of Task to be completed
- implement Reduxtoolkit. for checkout and payment 
- design a successful payment ui component

## How should this be manually tested?
- clone the repo `git clone https://github.com/atlp-rwanda/ecommerce-app-champions-fe.git`
- checkout to ` ft-create-payment-cp-92`
- create `.env `file and paste variables from `.env.example`
- install the packages by `npm install`
- start the server  `npm run dev`
-  login as `buyer`
- add product to `cart` 
- click on `checkout button`

<img width="815" alt="Screenshot 2023-05-14 at 09 17 06" src="https://github.com/atlp-rwanda/ecommerce-app-champions-fe/assets/89464364/31727484-4fb9-4a40-a084-060afcafd4bd">

## Trello card 92

oliviertech212 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [A buyer should be able to checkout and pay](https://trello.com/c/02jj0Oz0/92-a-buyer-should-be-able-to-checkout-and-pay)

Luciefifi reviewed (commented):
can you please give us more details about testing this???? and you are paying for something but the quantity of that item is not changing yet you payed for some quantity of that product

Shumba250 reviewed (changes requested):
work on the success page, it would be better if you design a detailed page or a receipt that contains payment info instead of a json

oliviertech212 committed:
feat(landingpage):create landing page
-design UI
-implememnt responsiveness
[WIP ft-create-landing-page]

wakaflorien committed:
Merge pull request #28 from atlp-rwanda/ft-add-to-cart-cp-86

#86 Feature : Add an item to cart

sostene12 committed:
ft(single-product): feature single product

- implement single product page

[finished ft-single-product-cp-99]

sostene12 committed:
ft(recommended-products): feature recommended products

- implement get recommened products
- implement their styles and responsiveness

[Finishes ft-recommended-products-cp-94]

oliviertech212 committed:
feat(landingpage)
-rebase to recommended products
[Finishes rebase]

oliviertech212 committed:
rebase to landingpage
